backend/internal/gateway/openai_compat_test.go

```
@@ -0,0 +1,475 @@
+	server := httptest.NewServer(f.gateway)
+	defer server.Close()
+	req, err := http.NewRequest(http.MethodPost, server.URL+"/v1/chat/completions", strings.NewReader(`{"model":"gateway-model","stream":true}`))
+	if err != nil {
+		t.Fatal(err)
+	}
+	req.Header.Set("Authorization", "Bearer "+f.secret)
+	resp, err := server.Client().Do(req)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)